Project1/String.h

```diff
@@ -0,0 +1,22 @@
+#include <exception>
+
+class StringIndexException : public std::exception {
+	using std::exception::exception;
+};
+
+class String {
+public:
+
```

Comment:
What happens if I write the following code:
{
  String s1 ("Hello world");
  String s2 = s1;
}
// more code

Read:
https://en.cppreference.com/w/cpp/language/rule_of_three